config/__init__.py

```diff
@@ -138,8 +138,9 @@ def convert_milestones(milestones_content):
 # creating an issue.
 EXTRA_LABELS = [
     'type-media',
+    'type-stylo',
+    'type-webrender-enabled',
     'type-webvr',
```

Comment:
I don't know if it's on purpose. The trailing comma doesn't matter. It's not improper both in style and validity, just noticing because it was not like this just before.